contentlayer.config.js

```diff
@@ -62,15 +62,15 @@ const computedFields = {
             },
           },
         ],
-        [
-          rehypeAutolinkHeadings,
-          {
-            properties: {
-              className: ["subheading-anchor"],
-              ariaLabel: "Link to section",
-            },
-          },
-        ],
+        // [
```

Comment:
Commented code wanted?

Reply:
This I want to keep, I may want to revisit this soon after the hackathon